Header header
ObjectInfo[] object_info
Relation[] relations
SimpleRelation[] simple_relations